src/garage/torch/algos/maml.py

```diff
@@ -165,8 +165,8 @@ def _train_value_function(self, paths):
 
         obs = np.concatenate([path['observations'] for path in paths], axis=0)
         returns = np.concatenate([path['returns'] for path in paths])
-        obs = torch.from_numpy(obs)
-        returns = torch.from_numpy(returns)
+        obs = torch.from_numpy(obs).float()
```

Comment:
please explain this change. what component provided a non-float return or observation?

Reply:
When I run the testcase `examples/torch/maml_trpo_ml10.py` I got the error

```
RuntimeError: Expected object of scalar type Float but got scalar type Double for argument #2 'mat1' in call to _th_addmm
```

So I set the tensor type in to float

Reply:
what is the upstream cause? @naeioi 

we can't just insert `float()` whenever something inconvenient happens.

Reply:
This looks like the type of `observations` in paths is float64 (which is double).
If we use `torch.Tensor`, the type changes to float32. (just like VPG)
However, MAML use `torch.from_numpy` and this maintain the dtype of numpy array.
So I changed the code to `torch.Tensor(obs)` instead of `torch.from_numpy(obs)`